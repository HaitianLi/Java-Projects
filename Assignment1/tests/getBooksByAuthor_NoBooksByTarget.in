add collection books.csv
author Haitian Li
exit