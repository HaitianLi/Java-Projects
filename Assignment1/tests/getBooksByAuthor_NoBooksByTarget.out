user: 17 books successfully added.

user: No books by Haitian Li.

user: Ending Library process.
